src/osx/webrequest_urlsession.mm

```diff
@@ -190,8 +190,10 @@ - (void)URLSession:(NSURLSession *)session
     // Set request headers
     for (wxWebRequestHeaderMap::const_iterator it = m_headers.begin(); it != m_headers.end(); ++it)
     {
-        [req setValue:wxCFStringRef(it->second).AsNSString() forHTTPHeaderField:
-         wxCFStringRef(it->first).AsNSString()];
+        // TODO: URLSession does not support multiple headers with the same name.
```

Comment:
FWIW there apparently is `[NSURLSession addValue:forHTTPHeaderField:]`, so you could try using it here if you're motivated.